packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/InAppPurchasePluginTests.m

```diff
@@ -441,6 +637,116 @@ - (void)testRemovePaymentQueueDelegate {
   }
 }
 
+- (void)testHandleTransactionsUpdated {
+  NSDictionary *transactionMap = @{
+    @"transactionIdentifier" : @"567",
+    @"transactionState" : @(SKPaymentTransactionStatePurchasing),
+    @"payment" : [NSNull null],
+    @"error" : [FIAObjectTranslator getMapFromNSError:[NSError errorWithDomain:@"test_stub"
+                                                                          code:123
+                                                                      userInfo:@{}]],
+    @"transactionTimeStamp" : @([NSDate date].timeIntervalSince1970),
+  };
+
+  InAppPurchasePlugin *plugin = [[InAppPurchasePlugin alloc] initWithReceiptManager:nil];
+  FlutterMethodChannel *mockChannel = OCMClassMock([FlutterMethodChannel class]);
+  plugin.transactionObserverCallbackChannel = mockChannel;
+  OCMStub([mockChannel invokeMethod:[OCMArg any] arguments:[OCMArg any]]);
+
+  SKPaymentTransactionStub *paymentTransaction =
+      [[SKPaymentTransactionStub alloc] initWithMap:transactionMap];
+  NSArray *array = [NSArray arrayWithObjects:paymentTransaction, nil];
+  NSMutableArray *maps = [NSMutableArray new];
+  [maps addObject:[FIAObjectTranslator getMapFromSKPaymentTransaction:paymentTransaction]];
+
+  [plugin handleTransactionsUpdated:array];
+  OCMVerify(times(1), [mockChannel invokeMethod:@"updatedTransactions" arguments:[OCMArg any]]);
```

Comment:
This is that one channel that wasn't replaced by pigeon right?

Reply:
Yeah, I just left it for now. But still wanna make sure its working correctly lol